_data/simple-icons.json

```diff
@@ -2005,6 +2005,11 @@
             "hex": "FF9400",
             "source": "http://design.firefox.com/photon/visuals/product-identity-assets.html"
         },
+        {
+            "title": "Musescore",
+            "hex": "1A70B8",
```

Comment:
Where did you get this colour from @foo-dogsquared? When I open the `.eps` in Gimp I'm getting `#067bb6` and on their site I'm also unable to find the colour you specified :thinking:

Reply:
I got the color from the [high resolution PNG](https://s3.amazonaws.com/s.musescore.org/about/images/design_MU3/musescore_sticker+11%403x.png) and the [PDF](https://s3.amazonaws.com/s.musescore.org/about/images/design_MU3/musescore_sticker+11.pdf). 
They both got the same result.
I've also tested it in the Firefox color dropper and in Inkscape.  
Seems getting the hex consistently so I go with that.

Reply:
It makes sense they all give the same result as they're all the same PNG (probably). Unfortunately rasterized images (such as PNG) or prone to color changes due to optimization. Since the `.eps` format doesn't have this problem I think that is a better source for the color. Would you mind updating it?

Reply:
I still get the color `#1A70B8` with Inkscape (I made sure it is opened with the EPS). 
Unfortunately, I cannot test this on GIMP due to some corrupted plug-in. 

In any case, can I suggest we settle this with someone testing this for additional input?

Reply:
In that case I'm fine with this value. If it turns out to be the incorrect choice it is easy to update this in the future anyway. 😉